projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/abstract-form-field.component.ts

```diff
@@ -46,6 +47,7 @@ export abstract class AbstractFormFieldComponent {
 
   protected addValidators(caseField: CaseField, control: AbstractControl): void {
     // No validators by default, override this method to add validators to the form control
+    FormValidatorsService.addValidators(caseField, control);
```

Comment:
Probably best not to do this, because it has quite wide impact. Instead can you just add the validators for the specific field types we are concerned about - text field and text area